php/test/Ice/binding/Client.php

```diff
@@ -53,7 +52,7 @@ function allTests($helper)
 
         $com->deactivateObjectAdapter($adapter);
 
-        $test3 = $test1->ice_uncheckedCast("::Test::TestIntf");
+        $test3 = $test1;
         test($test3->ice_getConnection() == $test1->ice_getConnection());
         test($test3->ice_getConnection() == $test2->ice_getConnection());
```

Comment:
Didn't we remove this part of the test in other mappings?

Reply:
Fixed.